assets/src/components/form/inputs/form-input.js

```diff
@@ -48,7 +48,7 @@ HTML5_INPUT_TYPES.push( 'iconButton' );
  * @param {Function} changeListener
  * @return {string} rendered date name form row
  */
-export class FormInput extends Component {
+class FormInput extends Component {
```

Comment:
What about refactoring this to functional component?

Reply:
ya that component is a bit of a hot mess, but didn't feel like tackling it atm